magellan-legacy/src/main/java/com/wealthfront/magellan/Navigator.kt

```diff
@@ -70,32 +77,8 @@ public class Navigator internal constructor(
     delegate.goTo(navigable, magellanTransition)
   }
 
-  public fun hideNow(navigable: NavigableCompat) {
-    hide(navigable, NoAnimationTransition())
-  }
-
-  @JvmOverloads
-  public fun hide(
-    navigable: NavigableCompat,
-    overrideMagellanTransition: MagellanTransition? = null
-  ) {
-    navigate(BACKWARD) { backStack ->
-      val backStackItem = backStack.find { it.navigable == navigable }
-        ?: throw IllegalStateException("Cannot find navigable (${navigable::class.java.simpleName}) in the backstack!")
-      backStack.remove(backStackItem)
-      NavigationEvent(navigable, overrideMagellanTransition ?: ShowTransition())
-    }
-  }
-
-  @JvmOverloads
-  public fun hide(magellanTransition: MagellanTransition? = null) {
-    navigate(BACKWARD) { backStack ->
-      NavigationEvent(backStack.pop().navigable, magellanTransition ?: ShowTransition())
-    }
-  }
-
```

Comment:
Don't we need these hide() functions for compatibility?

Reply:
I moved all our hides to just use goBack instead. This way you have just one way of navigating and navigating back is automatically handled for you.

Reply:
Ooooh sweet :)